system-tests/project-fixtures/svelte/cypress/support/component.js

```diff
@@ -0,0 +1,28 @@
+// ***********************************************************
+// This example support/component.js is processed and
+// loaded automatically before your test files.
+//
+// This is a great place to put global configuration and
+// behavior that modifies Cypress.
+//
+// You can change the location of this file or turn off
+// automatically serving support files with the
+// 'supportFile' configuration option.
+//
+// You can read more here:
+// https://on.cypress.io/configuration
+// ***********************************************************
+
+// Import commands.js using ES2015 syntax:
+import "./commands";
```

Comment:
/nitpick: `commands` file looks to be empty, could we just remove it and this import?

Reply:
Addressed in [b8cd5fe](https://github.com/cypress-io/cypress/pull/23553/commits/b8cd5fe1702b153d9d59a43775dffa63a5442178)